examples/employee-app/pom.xml

```diff
@@ -37,10 +37,13 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>io.helidon.integrations.db</groupId>
+            <artifactId>ojdbc</artifactId>
+        </dependency>
         <dependency>
             <groupId>com.oracle.database.jdbc</groupId>
-            <artifactId>ojdbc8-production</artifactId>
-            <type>pom</type>
+            <artifactId>ucp</artifactId>
```

Comment:
`ojdbc8-production` brings drivers and connection pooling both. Since we are replacing drivers to come via `io.helidon.integrations.db:ojdbc`, we still need to bring connection pooling for that to be equal as before.